packages/ui/src/components/va-pagination/VaPagination.vue

```diff
@@ -304,12 +292,68 @@ export default defineComponent({
       opacity: 0.4;
     }
 
-    &--outline.va-button--disabled {
-      opacity: 1;
+    &.va-button--disabled {
+      opacity: 0.8;
```

Comment:
Doesn't here will fit the `va-disabled` global mixin?

Reply:
It's already used in `VaButton` which is used by `VaButtonGroup` which is used by `VaButton` :).

But i just need to overwrite opacity, provided by mentioned mixin.

Reply:
> Current page number is barely visible

Will move `VaButton` out of `VaButtonGroup` can't set separate button state without it. Moreover, no benefits, only problems.